src/app/Base.tsx

```
@@ -51,42 +52,43 @@ export const Base : React.FC<BaseProps> = ({
+                            path={path}
+                            cidade={cidade}
+                            setCidade={setCidade}
+                            width={width}
+                            res={res}
+                            active={active}
+                            setMode={setMode}
+                        />
+                    }
+                    <div 
+                    style={{
+                        paddingTop: width > 1000  ? "76px" :  path == '/' ? "0px" : path == '/apoio' ? "0px" :"30px",
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)